reward/staking_manager.go

```diff
@@ -225,6 +226,15 @@ func getStakingInfoFromAddressBook(blockNum uint64) (*StakingInfo, error) {
 	}
 
 	caller := backends.NewBlockchainContractBackend(stakingManager.blockchain, nil, nil)
+	code, err := caller.CodeAt(context.Background(), addressBookContractAddress, nil)
+	if err != nil {
+		return nil, fmt.Errorf("failed to call AddressBook contract. root err: %s", err)
```

Comment:
`failed to retrieve code of AddressBook contract.` may be better, but the current one also looks fine.

Reply:
Fixed.